src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec

```diff
@@ -6,6 +6,8 @@
     <description>
         Contains the build task and targets used by MSBuild to run the C# and VB compilers.
         Supports using VBCSCompiler on Windows.
+
+        Source Code $commitUrl$
```

Comment:
The link works with a shortened commit hash, might make it easier on the eyes. Even better, we can get NuGet to recognize a link of this form and provide a friendly display form like dotnet/roslyn@5fbdd5c.